test/integration/framework/apiserver.go

```diff
@@ -56,16 +61,28 @@ func RunControlPlane(t *testing.T) (*rest.Config, StopFunc) {
 		t.Logf("Found CRD with name %q", crd.Name)
 	}
 	patchCRDConversion(crds, webhookOpts.URL, webhookOpts.CAPEM)
-	// environment variables
+
 	env := &envtest.Environment{
 		AttachControlPlaneOutput: true,
 		CRDs:                     crdsToRuntimeObjects(crds),
 	}
+
 	config, err := env.Start()
 	if err != nil {
 		t.Fatalf("failed to start control plane: %v", err)
 	}
-	// TODO: configure Validating and Mutating webhook
```

Comment:
This doesn't appear to configure the mutating webhook yet? Ideally we would, as mutation happens before validation meaning a request that is invalid (ie missing a required field) could be made valid by a mutating hook.

If it's not needed for now, keeping part of this comment would be good 😀

Reply:
Added the mutating one too as it is very similar even though the test may not fully depend on it right now